repository/user.go

```diff
@@ -44,6 +44,11 @@ func (r *UserRepository) Read(ctx context.Context, userID uuid.UUID) (svcmodel.U
 		resp.CreatedAt.Time,
 		resp.UpdatedAt.Time,
 	)
+	if err != nil {
+		return svcmodel.User{}, dberrors.NewToSvcModelErr(err)
+	}
```

Comment:
```
u, err := svcmodel.ToUser(
...
)
```

Returns service error, but I want to return a repository error indicating that the database data could not be converted to the service model (with details provided by that service error).